stream-chat-android-offline/src/test/java/io/getstream/chat/android/offline/extensions/ChannelExtensionsTest.kt

```diff
@@ -11,13 +10,16 @@ import org.junit.jupiter.api.Assertions
 import org.junit.jupiter.api.Test
 import java.io.File
 import java.util.Date
+import kotlin.reflect.javaType
+import kotlin.reflect.typeOf
 
 internal class ChannelExtensionsTest {
 
+    @OptIn(ExperimentalStdlibApi::class)
     @Test
     fun `When apply pagination Should not throw any exception`() {
         val channelsFile = File(this.javaClass.classLoader!!.getResource("channels.json").toURI())
-        val type = object : TypeToken<List<Channel>>() {}.type
+        val type = typeOf<List<Channel>>().javaType
```

Comment:
Tiny improvement to use fewer Gson APIs in this test - will need to be revisited later to see how we can parse this JSON with Moshi while being in the offline module